exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -99,48 +105,44 @@ public void Roll(int roll)
                 Console.WriteLine(players[currentPlayer]
                         + "'s new location is "
                         + places[currentPlayer]);
-                Console.WriteLine("The category is " + CurrentCategory());
+                Console.WriteLine("The category is " + SubjectOfQuestion());
                 AskQuestion();
             }
 
         }
 
         private void AskQuestion()
         {
-            if (CurrentCategory() == "Pop")
+            if (SubjectOfQuestion() == POP_CATEGORY)
             {
                 Console.WriteLine(popQuestions.First());
                 popQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Science")
+            if (SubjectOfQuestion() == SCIENCE_CATEGORY)
             {
                 Console.WriteLine(scienceQuestions.First());
                 scienceQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Sports")
+            if (SubjectOfQuestion() == SPORTS_CATEGORY)
             {
                 Console.WriteLine(sportsQuestions.First());
                 sportsQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Rock")
+            if (SubjectOfQuestion() == "Rock")
             {
                 Console.WriteLine(rockQuestions.First());
                 rockQuestions.RemoveFirst();
             }
         }
 
 
-        private string CurrentCategory()
+
+        private string SubjectOfQuestion()
```

Comment:
Using both subject & category to refer to the same thing might get confusing. I personally felt comfortable with the original name, but if you want to emphasise it's a result relating to a question still perhaps CategoryOfQuestion would work